### PR TITLE
fixed routes, replaced emotion theme with MUI provider, routes, heade…

### PR DESCRIPTION
**Pull request by NastyaAndreeva**
…r, modal for email, switchers markup, table markup

### CODE REVIEW

File: src/components/DashboardNavigation/CalendarView/CalendarView.styled.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      left: 0,
+      top: 0,
+      backgroundRepeat: 'no-repeat',
+      backgroundPosition: 'center',
+    },
+  },
+  '& .MuiSwitch-track': {
+    opacity: 1,
+    backgroundColor: theme.palette.mode === 'dark' ? '#8796A5' : '#aab4be',
+    borderRadius: 20 / 2,
```
Comment (Avinamor) [inline src/components/DashboardNavigation/CalendarView/CalendarView.styled.ts]: Why did you write 20 / 2, not just 10?

File: src/components/DashboardNavigation/CalendarView/CalendarView.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { FC } from 'react';
+import { MaterialUISwitch } from './CalendarView.styled';
+
+const CalendarView: FC = () => {
+  const isChecked = true;
```
Comment (Avinamor) [inline src/components/DashboardNavigation/CalendarView/CalendarView.tsx]: it's a redundant variable.

File: src/components/DashboardNavigation/DashboardNavigation.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import { AiOutlinePlus } from 'react-icons/ai';
+import Switcher from './Switcher';
+import CalendarView from 'components/DashboardNavigation/CalendarView';
+import {
+  NavigationContainer,
+  AddNewButton,
+  LeftSideNavigation,
+} from './DashboardNavigation.styled';
```
Comment (Avinamor) [inline src/components/DashboardNavigation/DashboardNavigation.tsx:9]: add space below

File: src/components/DashboardNavigation/DashboardNavigation.styled.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+import { NavLink } from 'react-router-dom';
+
+export const NavigationContainer = styled.div`
+  padding: 15px;
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-between;
+`;
```
Comment (Avinamor) [inline src/components/DashboardNavigation/DashboardNavigation.styled.ts:9]: add empty space below

File: src/components/DashboardTable/DashboardTable.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  notes: string
+) {
+  return { abscenceId, type, startDate, endDate, notes };
+}
+
+const rows = [
+  createData('1', 'regular', '03/12/2022', '17/12/2022', 'regular'),
+];
+
+const DashboardTable: FC<DashboardTableProps> = ({ isActual }) => {
```
Comment (Avinamor) [inline src/components/DashboardTable/DashboardTable.tsx]: this prop is not used. Please don't leave unused props. 
If you plan to use it in the future, create it and use it in the future or now, but don't leave them like that.

File: src/components/DashboardTable/index.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+export {default} from "./DashboardTable"
```
Comment (Avinamor) [inline src/components/DashboardTable/index.ts]: empty line, please be more attentive

**Review by Avinamor — APPROVED** [review]

File: src/components/DashboardNavigation/CalendarView/CalendarView.styled.ts (hunk 1 above)
Comment (NastyaAndreeva) [reply src/components/DashboardNavigation/CalendarView/CalendarView.styled.ts]: Fixed

File: src/components/DashboardNavigation/CalendarView/CalendarView.tsx (hunk 2 above)
Comment (NastyaAndreeva) [reply src/components/DashboardNavigation/CalendarView/CalendarView.tsx]: Fixed

File: src/components/DashboardNavigation/DashboardNavigation.tsx (hunk 3 above)
Comment (NastyaAndreeva) [reply src/components/DashboardNavigation/DashboardNavigation.tsx:9]: Done

File: src/components/DashboardTable/DashboardTable.tsx (hunk 5 above)
Comment (NastyaAndreeva) [reply src/components/DashboardTable/DashboardTable.tsx]: Fixed

File: src/components/DashboardTable/index.ts (hunk 6 above)
Comment (NastyaAndreeva) [reply src/components/DashboardTable/index.ts]: Fixed

**Pull request merged**